src/components/Modal.tsx

```diff
@@ -116,7 +115,10 @@ class Modal extends React.Component<ModalProps, any> {
 			>
 				<ModalBackdrop z={8888} bg="rgba(0,0,0,0.4)" top right bottom left />
 				<ModalSizer>
-					<ModalPanel w={w || width || DEFAULT_MODAL_WIDTH} onClick={stopEvent}>
+					<ModalPanel
+						w={w || width || DEFAULT_MODAL_WIDTH}
+						onClick={(e: Event) => e.stopPropagation()}
```

Comment:
It'd be nice to avoid arrow functions in render. Some background here: https://medium.freecodecamp.org/why-arrow-functions-and-bind-in-reacts-render-are-problematic-f1c08b060e36.

Not a big deal, but since we might want to do this in lots of places, imo it'd be slightly better to avoid this and simplify things with a single `stopPropagation` function in util, and just use that everywhere.